### DISCUSSION TITLE
Multiple language servers don't get attached to buffer

### DISCUSSION THREAD

**Discussion by borisbrodski** [mason-org/mason.nvim#1131] [Q&A]
Hi,

I'm using Mason on Windows with AstroVim. My configuration:

* Windows 10
* NeoVim v0.8.3
* AstroNVim v3.3.2

I expirience the problem, that different language servers don't get attached to the corresponding buffers:

* bashls (to a buffer with *.sh extension)
* jdtls (to a buffer with *.java extension)

On the other hand following language servers get attached and are working well:

* lua_ls
* lemminx
* yamlls
* jsonls
* tsserver

MasonLog is empty.

How can I investigate the problem?

Thank you!

Cheers,
Boris

**Comment by avegancafe** [comment]
Does `:LspLog` yield anything? `:MasonLog` I believe is for the install logs
